configs/allow_namespaces_from_config.yml

```diff
@@ -0,0 +1,8 @@
+apiVersion: v1
```

Comment:
🤔 is its only intended use in tests? if so I think it shouldn't be in the config directory.